### PR TITLE
Updated README

### LINKED ISSUE #177: Add Documentation in README.
- [x] Add UI Screens 
- [x] Add Instructions to register
- [x] Add Details about Profile Page, Favorite events
- [x] Add Team Data

### PR DESCRIPTION
**Pull request by Abhinav-ark**
Adding Documentation to README (In Progress...)

Closes #177

### CODE REVIEW

**Comment by Thanus-Kumaar** [comment]
Made the necessary changes to the readme.md, do check @Ashrockzzz2003 @Abhinav-ark

**Review by Ashrockzzz2003 — CHANGES_REQUESTED** [review]
Awesome work @Thanus-Kumaar 
Let's add links to other repos here. 

Backend, Mobile app, admin app, qr scan ... etc

File: README.md [hunk 1]
```diff
@@ -1,38 +1,131 @@
...
+
+Follow the event registration process for individual events.  
+
+https://github.com/user-attachments/assets/bae59551-9701-4790-b5d0-8a890954fcfb
+
+---
+
+## 💳 View Transactions
+
+Manage and view your transactions effortlessly.
```
Comment (Abhinav-ark) [inline README.md]: The Video is to `View Registration and Submit Feedback`. Transactions Will be In Profile Page.

**Review by Abhinav-ark — COMMENTED** [review]
@Thanus-Kumaar  `Profile Page` and `View Team Event Registation Details` Are Missing.

**Review by Ashrockzzz2003 — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,38 +1,156 @@
...
+
+
+
+https://github.com/user-attachments/assets/ed589656-b210-4b45-a697-c43e7c216e2d
+
+
+
+---
+
+## 💳 View Transactions
```
Comment (Abhinav-ark) [inline README.md:52]: This is not just transactions. Change Emoji also.

File: README.md [hunk 3]
```diff
@@ -1,38 +1,156 @@
...
+
+## 📝 Event/Workshop Registration
+
+Follow the event registration process for individual events.  
+
+https://github.com/user-attachments/assets/bae59551-9701-4790-b5d0-8a890954fcfb
+
+---
+
+## Profile Page
```
Comment (Abhinav-ark) [inline README.md:40]: Why does this h2 not have an emoji? Use 👤.

File: README.md [hunk 4]
```diff
@@ -1,38 +1,156 @@
...
+
+Visit your profile to update your details and view registered events.
+
+
+
+https://github.com/user-attachments/assets/ed589656-b210-4b45-a697-c43e7c216e2d
```
Comment (Abhinav-ark) [inline README.md:46]: View Transactions functionality not showcased in profile page.

**Review by Abhinav-ark — COMMENTED** [review]
Look into following changes.

**Pull request merged**